content/ja/tightly-coupled-architectures.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 密結合アーキテクチャ
+status: 完了
+category: プロパティ
+tags: ["基礎", "アーキテクチャ", "プロパティ"]
+---
+
+密結合アーキテクチャは、アプリケーション構成要素の多くが互いに依存しているようなアーキテクチャを指します
+(その逆は[疎結合アーキテクチャ](/loosely-coupled-architecture/)と呼ばれます)。
+密結合であることは、ある構成要素に対する変更がその他の構成要素に影響を与えうることを意味します。
+一般的に、密結合アーキテクチャはより疎結合なアーキテクチャと比較して簡単に実装できますが、カスケード障害と呼ばれる連鎖的な障害に対してシステムがより弱くなる可能性があります。
+また、しばしば構成要素を協調してロールアウトする必要が生じるため、開発者の生産性に対する足かせとなる可能性があります。
+
+密結合なアプリケーションアーキテクチャはかなり古典的なアプリケーション構築方法です。
+[マイクロサービス](/microservices/)開発のベストプラクティス全てには必ずしも整合しませんが、
+密結合アーキテクチャは特定の場合に有用です。
+密結合アーキテクチャでの開発は、実装がより簡潔で早くなる傾向があり、
+[モノリシックアプリケーション](/monolithic-apps/)とよく似て初期開発サイクルを加速させる可能性があります。
```

Comment:
ditto

Reply:
Fixed in [cf90e8c](https://github.com/cncf/glossary/pull/2365/commits/cf90e8c7ec15ac133c6defe0e2a9882e4863be09).